lib/chef/platform/rebooter.rb

```diff
@@ -35,7 +35,9 @@ def reboot!(node)
 
           cmd = if Chef::Platform.windows?
                   # should this do /f as well? do we then need a minimum delay to let apps quit?
-                  "shutdown /r /t #{reboot_info[:delay_mins] * 60} /c \"#{reboot_info[:reason]}\""
+                  # Use explicit path to shutdown.exe, to protect against https://github.com/chef/chef/issues/5594
+                  windows_shutdown_path = "#{ENV['SystemDrive']}/Windows/System32/shutdown.exe"
```

Comment:
Can you use `#{ENV['SYSTEMROOT']/system32` instead? This would ensure this works if windows is installed to a directory named something other than 'windows'.